examples/mobile/HomeApp/js/app.js

```diff
@@ -49,6 +52,16 @@
 		loadWeatherJS();
 	}
 
+	function loadWebClipList() {
+		const requestURL = "api/webcliplist";
```

Comment:
as Tomasz said, it contains CORS issues.
If the URL is run locally, you need to consider the issue.

Reply:
ok, I understand case although this is specific use case.

app is served always with API on the same port so by default no issue occurs. This is only in case when  API will be served separately from app, but this is not our use case I think

Reply:
ok